compliance/aws/disallowed_regions/aws_disallowed_regions.pt

```diff
@@ -205,17 +198,29 @@ end
 policy "pol_list_aws_instances" do
   validate $ds_list_aws_instances do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} AWS Instances in Disallowed regions"
-    detail_template <<-EOS
-# AWS Instance Details
-| Instance ID | Instance State | Region | Instance Type | Tags |
-| ----------- | -------------- | ------ | ------------- | ---- |
-{{ range data -}}
-| {{.instance_id}} | {{.instance_state}}  | {{.region}} | {{.instance_type}} | {{.tagKeyValue}} |
-{{ end -}}
-    EOS
-    escalate $esc_report_disallowed_instances
-    escalate $esc_terminate_instances_disallowed_region
+
+    export do
+      resource_level true
+      field "id" do
```

Comment:
This doesn't HAVE to change, but it's less changes in the template to use path here instead of changing the instance_id everywhere in the template.

>       field "id" do
>        label "Instance ID"
>        path jmes_path(item, "instance_id")
>      end